### ISSUE TITLE
Flying Carpet misaligns on large screens until explicit width is added by amp runtime

### ISSUE THREAD

**Issue by paraleadogg** [ampproject/amphtml#25393]
*** Duplicate of this issue [here](https://github.com/ampproject/amp-wp/issues/3680) - think I may have opened in the wrong place. Feel free to delete as necessary

Hi Guys,

Spotted an issue with the Flying Carpet component that is actually visible on your own demo page here:

https://amp.dev/documentation/examples/components/amp-fx-flying-carpet/#flying-carpet-for-ads

If the viewport is over 1440px wide the ad in the flying carpet is aligned over to the right as there is no width set on the `.i-amphtml-fx-flying-carpet-container` fixed element 

The width is only set when you resize the screen

## Expected Behaviour

The width on the fixed container should be set on load so the ad within the flying carpet is center aligned in all viewports

## Steps to reproduce

1. Go to https://amp.dev/documentation/examples/components/amp-fx-flying-carpet/#flying-carpet-for-ads on a screen wider than 1440px wide
2. Observe the ad inside the flying carpet is aligned to the right-hand side (possibly off-screen)
3. Slowly resize your screen
4. Once your screen size drops below 1440px width the width attribute is added to the `<div class="i-amphtml-fx-flying-carpet-container"` by the amp runtime and the ad correctly jumps to the correct centered position

## Screenshots

![before](https://user-images.githubusercontent.com/10920375/68039290-bbbc1a80-fcc3-11e9-978d-c97938252fc1.JPG)

![after](https://user-images.githubusercontent.com/10920375/68039322-cd9dbd80-fcc3-11e9-8fb8-9a0dd4a5d233.JPG)

## Additional context

 - WordPress version: 5.2.4
 - Plugin version: 1.4.0
 - Browser: Firefox and Chrome

**Comment by mrjoro** [comment]
I transferred this to amphtml, and /cc @ampproject/wg-ui-and-a11y @ampproject/wg-ads

**Comment by jeffjose** [comment]
Over to @lannka

**Comment by calebcordry** [comment]
@paraleadogg Thanks for the report. I was able to repro on the amp.dev page and locally. It does not appear to be an ads specific issue as the `amp-img` example right below also has the same behavior. 

We are writing the inline width `onMeasureChanged` here:
https://github.com/ampproject/amphtml/blob/595c167c0f1bde926a2cbe63f459cc2234f8110b/extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js#L100-L103

Doing this from the beginning sounds reasonable to me, but I will let @ampproject/wg-ui-and-a11y make the call. What do you all think?

**Comment by nainar** [comment]
@sparhami but yeah that makes sense to me.
